components/n-ui/roe/can-i-please-get-a-puppy.js

```diff
@@ -0,0 +1,33 @@
+const canIPleaseGetAPuppy = () => {
+	const puppyId = 'canIPleaseGetAPuppy';
+	const isPuppyArticle = document.documentElement.getAttribute('data-content-id') === 'b1a49898-2c44-11e8-a34a-7e7563b0b0f4';
```

Comment:
could this just be on _every_ article? 🙈

Reply:
haha! sure